hack/errata.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+# https://datagrepper.engineering.redhat.com/
+# https://mojo.redhat.com/docs/DOC-1072237
+
+import argparse
+import codecs
+import datetime
+import json
+import logging
+import time
+import urllib.parse
+import urllib.request
+
+
+logging.basicConfig(level=logging.DEBUG)
+_LOGGER = logging.getLogger()
+
+
+def load(path):
+    try:
+        with open(path, 'r') as f:
+            return json.load(f)
+    except FileNotFoundError:
+        return {}
+
+
+def save(path, cache):
+    with open(path, 'w') as f:
+        json.dump(cache, f, sort_keys=True, indent=2)
+
+
+def run(poll_period=datetime.timedelta(seconds=3600), cache=None, webhook=None, **kwargs):
+    next_time = datetime.datetime.now()
+    while True:
+        _LOGGER.debug('poll for messages')
+        for message in poll(period=2*poll_period, **kwargs):
+            if cache and message['fulladvisory'] in cache or 'bug fix update' not in message['synopsis']:
```

Comment:
`bug fix update` is a cheap hack.  We should really be comparing `fulladvisory` like `RHBA-2020:0936-04` to Cincinnati URI like:

```console
$ curl -sH 'Accept:application/json' 'https://api.openshift.com/api/upgrades_info/v1/graph?channel=candidate-4.2' | jq -r '.nodes[] | select(.version == "4.2.26").metadata.url'
https://access.redhat.com/errata/RHBA-2020:0936
```

Although I'm not clear on where the `-04` suffix went.